assembly/std/index.ts

```diff
@@ -102,6 +103,103 @@ export function balanceOf(address: string): u64 {
   return env.balanceOf(address);
 }
 
+/**
+ * Check for key in datastore
+ *
+ * @param {StaticArray<u8>} key
+ *
+ * @return {bool} - true if key is present in datastore, false otherwise.
+ */
+export function hasOpKey(key: Uint8Array): bool {
+  let result = env.hasOpKey(key.buffer);
+  // From https://doc.rust-lang.org/reference/types/boolean.html &&
+  // https://www.assemblyscript.org/types.html
+  // we can safely cast from u8 to bool
+  return bool(result[0]);
+}
+
+/*
+ * Get data associated with the given key from datastore
+ *
+ * @param {StaticArray<u8>} key
+ *
+ * @return {StaticArray<u8>} - data as a byte array
```

Comment:
the documentation (teyturn type) of this function (and others) needs to be updated

Reply:
done